### PR TITLE
Fixed format issue in code

### PR DESCRIPTION
**Pull request by vgelani**
## This PR is a:
- [ ] Bug fix or improvement
 

## Summary
I have added coding format in missing document.

### CODE REVIEW

File: guides/v2.1/coding-standards/code-standard-demarcation.md [hunk 1]
```diff
@@ -471,7 +471,9 @@ jQuery('#{$htmlId}-suggest').treeSuggest({$selectorOptions});
 <li>Reduces the number of files to be modified.</li></ul>
 
 **Acceptable PHP file**
-<pre>
+
+{% highlight php startinline=true %}
```
Comment (shrielenee) [inline guides/v2.1/coding-standards/code-standard-demarcation.md]: For code highlighting where there isn't a preceding `php` in the code snippet (like in this instance) use the following format for code:

```php?start_inline=1
<php code without starting <php? tag>
```

**Comment by shrielenee** [comment]
Heya @vgelani - I've added a comment about the way we format code samples like the ones in this page. Please review, make the changes to all code samples in this page, and re-push for review!

File: guides/v2.1/coding-standards/code-standard-demarcation.md (hunk 1 above)
Comment (vgelani) [reply guides/v2.1/coding-standards/code-standard-demarcation.md]: @shrielenee need to write _?start_inline=1_ text?

Comment (shrielenee) [reply guides/v2.1/coding-standards/code-standard-demarcation.md]: screen shot 2018-06-07 at 11 15 03 am

Comment (shrielenee) [reply guides/v2.1/coding-standards/code-standard-demarcation.md]: I had to reply with a screenshot because its difficult to escape formatting here!

**Comment by vgelani** [comment]
@shrielenee I have updated php code format. Please verify at once.

**Comment by shrielenee** [comment]
@vgelani - I updated some of the PHP formatting so you can see what I mean. Also, ensure that HTML code blocks start with a "``` html" and end with a "```".

**Comment by vgelani** [comment]
@shrielenee I have update HTML code formatting but I didn't use html code formatting for phtml code. I didn't work on it.

**Comment by shrielenee** [comment]
Ok I just checked, and it looks like the inline PHP code works for PHTML: ` ```php?start_inline=1 ` before the code and ` ``` ` after. Can you make that last change to the PHTML code and we should be ready to merge?

**Comment by vgelani** [comment]
@shrielenee I have updated code formatting for js, css and phtml. Please verfiy at once.

**Pull request merged**